.gitlab-ci.yml

```diff
@@ -73,45 +73,6 @@ build_puppy_agent-deb_x64:
   script:
     - inv -e agent.build --puppy
 
-#
-# integration_test
-#
-
-# run benchmarks on deb
-run_benchmarks-deb_x64:
```

Comment:
Can you just skip it (prefixing the name with a `.`) instead of deleting it ? I guess the long term plan is to keep it here, but it's failing for now.